translations/Marathi/general-rules.md

```diff
@@ -0,0 +1,40 @@
+# General Rules for Translation in Marathi
+
+## Overview
+
+* Marathi is a very versatile & easy-to-use language like English, so translations
+  are also easy.
+
+* Translations can be done in either way, formal or informal, but formal way is preferred
+  for corporate work.
+
+* The translations can be done in any person( first, second or third) according to the 
+  content one wants to translate.
+
+* Same is applicable for singular & plural, the translations depend upon the type of content.
+
+* The usage of verbs & nouns is similar to English, the type of the verb & the noun used changes
+  according to the construction of the sentence & the content, so there are no such rules 
+  for verbs & nouns.
+
+* The usage of particles( e.g. reflexives) again depends upon the sentence, so their is no 
+  restriction on their usage.
+
+* Slang words( especially abusive words) should not be used while translations.
+
+* Long Words are very uncommon in Marathi( especially Zulip-specific), so the translations
+  are easy & quick.
+* There is no specific way of addressing people in Marathi, it can be done in any way( formal 
```

Comment:
Can you please include some examples of the formal/preferred way and some of the informal?

Reply:
sure!